py/picca/blinding_a.py

```diff
@@ -1,71 +1,112 @@
 """This module defines some functions to blind data using strategy A"""
 import numpy as np
-
 from picca.constants import Cosmo, ABSORBER_IGM
+import sys, os  # requiered to mute Cosmo and dont print the
```

Comment:
Not necessary, same as functions blockPrint and enablePrint. I have modified the Cosmo class to not print the cosmology chosen when strategy A is used